code-challenges.test.js

```diff
@@ -15,35 +15,117 @@
 
```

Comment:
Good work on the interview questions and ruby challenges. Let's focus on the jest section and nailing down an approach you can use on the jest section. Keep practicing. I can see improvements